src/test/java/de/bonndan/nivio/assessment/kpi/LifecycleKPITest.java

```diff
@@ -24,8 +24,7 @@ public void setup() {
 
     @Test
     public void green() {
-        Item item = new Item();
-        item.setIdentifier("foo");
+        Item item = new Item(null, "foo");
```

Comment:
Adding a constructor for this case might increase readability of item construction, as the reader does not have to reason about `null`.

Reply:
In productive code the constructor is only used in ItemFactory, and there both args are given.